packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -138,9 +138,77 @@ class _DriverBinding extends BindingBase with SchedulerBinding, ServicesBinding,
 /// }
 /// ```
 ///
-void enableFlutterDriverExtension({ DataHandler? handler, bool silenceErrors = false, List<FinderExtension>? finders}) {
+/// Note: SomeCommand, SomeResult and SomeCommandExtension should be placed in
+/// different files to avoid `dart:ui` import issue.
+///
+/// ```dart
+/// class SomeCommand extends CommandWithTarget {
+///  SomeCommand(SerializableFinder finder, this.times, {Duration? timeout})
+///      : super(finder, timeout: timeout);
+///
+///  SomeCommand.deserialize(Map<String, String> json, DeserializeFinderFactory finderFactory)
+///      : times = int.parse(json['times']!),
+///        super.deserialize(json, finderFactory);
+///
+///  @override
+///  Map<String, String> serialize() {
+///    return super.serialize()..addAll(<String, String>{'times': '$times'});
+///  }
+///
+///  @override
+///  String get kind => 'SomeCommand';
+///
+///  final int times;
+///}
+///```
+///
+/// ```dart
+/// class SomeCommandResult extends Result {
+///   const SomeCommandResult(this.resultParam);
+///
+///   final String resultParam;
+///
+///   @override
+///   Map<String, dynamic> toJson() {
+///     return <String, dynamic>{
+///       'resultParam': resultParam,
+///     };
+///   }
+/// }
+/// ```
+///
+/// ```dart
+/// class SomeCommandExtension extends CommandExtension {
+///   @override
+///   String get commandKind => 'SomeCommand';
+///
+///   @override
+///   Future<Result> call(Command command, WidgetController prober, CreateFinderFactory finderFactory, CommandHandlerFactory handlerFactory) async {
+///     final SomeCommand someCommand = command as SomeCommand;
+///
+///     // Submit known [Command]s:
+///     for (int i = 0; i < someCommand.times; i++) {
```

Comment:
nit: flutter style says to use `index` instead of `i`, and to avoid `++`.
```suggestion
///     for (int index = 0; index < someCommand.times; index += 1) {
```

Reply:
OK